ui/src/components/admin/sermonNotes/AdminSermonNotesContainer.js

```diff
@@ -0,0 +1,362 @@
+import React, { useState, useEffect } from 'react';
+import { customAxios as axios } from '../../helpers/customAxios';
+import {
+  Container,
+  Heading,
+  Text,
+  Stack,
+  Button,
+  ListItem,
+  Flex,
+  Grid,
+  Spacer,
+  List,
+  Box,
+  Image,
+  HStack,
+  useToast,
+} from '@chakra-ui/react';
+import SermonNotesEditorModal from './SermonNotesEditorModal';
+import { MdOutlineVideoLibrary } from 'react-icons/md';
+import { BiDonateHeart } from 'react-icons/bi';
+import { IoPeopleOutline } from 'react-icons/io5';
+import { IoBookOutline } from 'react-icons/io5';
+import { FaRegCalendarAlt } from 'react-icons/fa';
+
+export default function AdminSermonNotesContainer(props) {
+  const { user } = props;
+  const toast = useToast();
+
+  // This useState is to open the Editor Modal
+  const [isEditorOpen, setIsEditorOpen] = useState(true);
+  const [actionOnEditor, setActionOnEditor] = useState('create');
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [sermonNotesList, setSermonNotesList] = useState([]);
+  const [editSermonNotesData, setEditSermonNotesData] = useState(null);
+
+  const fetchSermonNotes = async () => {
+    try {
+      const { data, status } = await axios.get('/api/sermon-notes-parent/get');
+      if (status !== 200) {
+        throw Error('There was an issue with the request');
+      }
+      setSermonNotesList(data);
+    } catch (err) {
+      console.log(err);
+      toast({
+        description:
+          'There was an issue with the request, please talk to t3ch support',
+        status: 'warning',
+        duration: 8000,
+        isClosable: true,
+      });
+    }
+  };
+
+  const isPublishDisabled = () => {
+    const aboveT3chPrivs = ['t3ch', 'admin', 'stewardship'];
+    if (aboveT3chPrivs.includes(user.accessType)) {
+      return false;
+    }
+    return true;
+  };
+
+  const onDelete = async (e) => {
+    try {
+      if (window.confirm('Are you sure you want to delete this sermon note?')) {
+        const { status } = await axios.put('/api/sermon-notes-parent/delete', {
+          sermonId: e.target.value,
+        });
+        if (status === 200) {
+          toast({
+            description: 'Sermon Notes has been deleted',
+            status: 'success',
+            duration: 8000,
+            isClosable: true,
+          });
+        }
+      }
+      await fetchSermonNotes();
+      setIsLoading(false);
```

Comment:
I think the reason why I do this, is because I don't want every time I load the page, all the buttons have the loading thingy, so I most prob will ignore this one